contracts/poap/src/msg.rs

```diff
@@ -0,0 +1,101 @@
+use cosmwasm_std::{Addr, Timestamp};
+use cw721_base::InstantiateMsg as Cw721InstantiateMsg;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct InstantiateMsg {
+    /// Address of who will have the right to administer the contract.
+    pub admin: String,
+    /// Address of who can call the [`ExecuteMsg::MintTo`] other then the admin.
+    pub minter: String,
+    /// Id of the CW721 contract to initialize together with this contract.
+    pub cw721_code_id: u64,
```

Comment:
Should we use `Uint64` as input and output instead? The normal JSON `u64` is represented as a `number`, but Amino JSON for cosmos ecosystem `u64` is represented as `string`. What do you think  @bragaz, @manu0466

Reply:
Yes, even if the signing with ledger is working with u64 it's better to use `Uint64` since can happen that a Javascript client does not represents correctly integer with more then 53 bits